docs/sources/source/speech_command/datasets.rst

```diff
@@ -0,0 +1,42 @@
+Datasets
+========
+
+.. _GoogleSpeechCommands_dataset:
+
+
+Google Speech Commands Dataset
+-----------------------------------
+
+The ability to recognize spoken commands with high accuracy can be useful in a variety of contexts.
+To this end, Google released the Speech Commands dataset (see :cite:`speech-recognition-dataset-warden2018speech`,
+which contains short audio clips of a fixed number of command words such as “stop”, “go”, “up”, “down”, etc spoken by a large number of speakers.
+To promote the use of the set, Google also hosted a Kaggle competition, in which the winning team attained a multiclass accuracy of 91%.
+
+We experimented with applying NeMo’sYes ill jus ASR classification models on mel spectrograms of the audio clips and found that they worked surprisingly well.
```

Comment:
Looks like a typo

Reply:
This and below issues fixed in latest commit